code/client/src/pages/Create.jsx

```diff
@@ -125,7 +133,7 @@ const Create = () => {
         root: ONEUtil.hexView(root),
         duration,
         effectiveTime,
-        lastResortAddress,
+        validLastResortAddress,
```

Comment:
Same issue as above

Reply:
Also, this goes into the state model and persistent storage. It should not use bech32 address